code/datums/mutantraces.dm

```diff
@@ -2246,6 +2246,7 @@ TYPEINFO(/datum/mutantrace/cow)
 				src.clothes_filters_active = !src.clothes_filters_active
 				boutput(src.mob, src.clothes_filters_active ? "Bovine-specific clothes filters activated." : "Disabled bovine-specific clothes filters.")
 				src.mob.update_clothing()
+				. = "<B>[src.mob]</B> readjusts [his_or_her(src.mob)] udders."
```

Comment:
`boutput` should work. No need to do `. =` either, just:

`boutput(usr, SPAN_NOTICE("You readjust your udders."))`

Reply:
Or visible_message depending on what you want

Reply:
This would ultimately lead to the same problem as before, the way emote code is written an emote needs some kind of return value for it to be considered valid, if it doesn't have one then it'll say "Unusable emote" (even if it works).

Reply:
> This would ultimately lead to the same problem as before, the way emote code is written an emote needs some kind of return value for it to be considered valid, if it doesn't have one then it'll say "Unusable emote" (even if it works).

Right. I see your concern. Honestly, this seems like more of a necessary refactor of emote code than an udder PR?
The real reason seems like that mutantraces use their own scuffed emote code and that should get fixed instead. This PR really just otherwise disguises a limitation.

Reply:
If a larger refactor is necessary as opposed to fixing this I'm just going to close this PR  --  that's beyond the scope of my current knowledge/time/abilities and would be better served being done by someone who understands it, I just wanted to fix the bug with what was available in code now